comictalker/comictalker.py

```diff
@@ -0,0 +1,200 @@
+            return issues_result
+        except NotImplementedError:
+            logger.warning(f"{self.talker.source_details.name} has not implemented: 'fetch_issues_by_volume'")
+            raise TalkerError(
+                self.talker.source_details.name,
+                4,
+                "The source has not implemented: 'fetch_issues_by_volume'",
+            )
+
+    # For CLI
+    def fetch_issue_data_by_issue_id(self, issue_id: int) -> GenericMetadata:
+        try:
+            issue_result = self.talker.fetch_issue_data_by_issue_id(issue_id)
+            return issue_result
+        except NotImplementedError:
+            logger.warning(f"{self.talker.source_details.name} has not implemented: 'fetch_issue_data_by_issue_id'")
+            raise TalkerError(
+                self.talker.source_details.name,
+                4,
+                "The source has not implemented: 'fetch_issue_data_by_issue_id'",
+            )
+
+    # For issueidentifer
+    def fetch_alternate_cover_urls(self, issue_id: int) -> list[str]:
+        try:
+            alt_covers = self.talker.fetch_alternate_cover_urls(issue_id)
+            return alt_covers
+        except NotImplementedError:
+            logger.warning(f"{self.talker.source_details.name} has not implemented: 'fetch_alternate_cover_urls'")
+            raise TalkerError(
+                self.talker.source_details.name,
+                4,
+                "The source has not implemented: 'fetch_alternate_cover_urls'",
+            )
+
+    # For issueidentifier
+    def fetch_issues_by_volume_issue_num_and_year(
+        self, volume_id_list: list[int], issue_number: str, year: str | int | None
+    ) -> list[ComicIssue]:
+        try:
+            issue_results = self.talker.fetch_issues_by_volume_issue_num_and_year(volume_id_list, issue_number, year)
+            return issue_results
+        except NotImplementedError:
+            logger.warning(
+                f"{self.talker.source_details.name} has not implemented: 'fetch_issues_by_volume_issue_num_and_year'"
+            )
+            raise TalkerError(
+                self.talker.source_details.name,
+                4,
+                "The source has not implemented: 'fetch_issues_by_volume_issue_num_and_year'",
+            )
+
+    def fetch_issue_cover_urls(self, issue_id: int) -> tuple[str | None, str | None]:
+        try:
+            cover_urls = self.talker.fetch_issue_cover_urls(issue_id)
+            return cover_urls
+        except NotImplementedError:
+            logger.warning(f"{self.talker.source_details.name} has not implemented: 'fetch_issue_cover_urls'")
+            raise TalkerError(
+                self.talker.source_details.name,
+                4,
+                "The source has not implemented: 'fetch_issue_cover_urls'",
+            )
+
+    # Master function to get issue cover. Used by coverimagewidget
+    def async_fetch_issue_cover_urls(self, issue_id: int) -> None:
```

Comment:
For the same reasons as `fetch_issue_cover_urls` this is never called and so we can remove